src/view/list-filter-view.js

```diff
@@ -1,38 +1,45 @@
 import AbstractView from '../framework/view/abstract-view.js';
 
-function createListFilterTemplate() {
+const createFilterItemTemplate = (filters) => {
+  const {name, filteredPoints} = filters;
+  const isDisabled = () => filteredPoints.length === 0;
+
   return (
-    `<form class="trip-filters" action="#" method="get">
-      <div class="trip-filters__filter">
-        <input id="filter-everything" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="everything">
-        <label class="trip-filters__filter-label" for="filter-everything">Everything</label>
-      </div>
-
-      <div class="trip-filters__filter">
-        <input id="filter-future" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="future">
-        <label class="trip-filters__filter-label" for="filter-future">Future</label>
-      </div>
-
-      <div class="trip-filters__filter">
-        <input id="filter-present" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="present">
-        <label class="trip-filters__filter-label" for="filter-present">Present</label>
-      </div>
-
-      <div class="trip-filters__filter">
-        <input id="filter-past" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="past" checked>
-        <label class="trip-filters__filter-label" for="filter-past">Past</label>
-      </div>
+    `<div class="trip-filters__filter">
+      <input id="filter-${name}"
+      class="trip-filters__filter-input  visually-hidden"
+      type="radio" name="trip-filter"
+      value="${name}"${isDisabled() ? 'disabled' : ''}>
+      <label class="trip-filters__filter-label"
+      for="filter-${name}">${name}</label>
```

Comment:
Примерно такое форматирование
```suggestion
      <input
        id="filter-${name}"
        class="trip-filters__filter-input  visually-hidden"
        type="radio"
        name="trip-filter"
        value="${name}"
        ${isDisabled() ? 'disabled' : ''}
      >
      <label class="trip-filters__filter-label" for="filter-${name}">${name}</label>
```